app/javascript/crayons/formElements/RadioButton/__stories__/radioButton.html.stories.jsx

```diff
@@ -2,7 +2,7 @@ import { h } from 'preact';
 import { Fieldset } from '../../../storybook-utilities/Fieldset';
 import '../../../storybook-utilities/designSystem.scss';
 
-export default { title: 'Components/Form Components/Radio Button/HTML' };
+export default { title: 'Components/Form Elements/Radio Button/HTML' };
```

Comment:
I don't think we need these anymore since we have the HTML copy/paste tab now. Doesn't have to be something we change in this PR though

Reply:
Yeah, these were all put in when Preact wasn't supported for the copy/paste tab. A separate PR makes sense for this.